### ISSUE TITLE
@kyle: figures for view-copies

### ISSUE THREAD

**Issue by nickeubank** [nickeubank/mids_coursera#43]
Hey @kylebradbury 

Could I get two things:

- A PNG for the first figure in [this](https://github.com/nickeubank/mids_coursera/blob/main/figure_prep/figures-views-and-copies.pdf) pdf that shows our figure conventions? I could pull but seems like you had an export trick and I want to keep resolutions etc. consistent. 
- Figures for when we get into views and copies in week 3 (in [this notebook](https://github.com/nickeubank/mids_coursera/blob/main/class_2/week_3/10_views_and_copies.ipynb) where there's currently just `[new image with referring arrow]`)? 

Thanks!

Nick

**Comment by nickeubank** [comment]
In all honesty, it may be good to have several more figures in that reading... any chance you could look it over and edit as you see fit? It's SO slippery...

**Comment by nickeubank** [comment]
> Added the image [here](https://github.com/nickeubank/mids_coursera/blob/main/class_2/week_3/img/notation-conventions.png). Feel free to place a reference where you'd like.

Integrated!

**Comment by kylebradbury** [comment]
Including the ones mentioned above, I've added about a dozen figures to lessons 2.3.10, 2.3.30, and 2.3.42 (where I'm listing them as Course.Week.Number) and I pushed the changes. Happy to edit them, but as I was going through a review of those sections it seemed. Please let me know if you like/hate those and happy to edit, or if there were any others for Course 2 that may be useful.

**Comment by nickeubank** [comment]
Amazing!

For 2.3.30, I'm wondering if it will be a little confusing, given people read from the top to the bottom of images, to have columns appear before rows:

![image](https://user-images.githubusercontent.com/9683693/170527235-0c09b3e8-82f0-4a8f-b479-d666e22d0a5d.png)

Thoughts on rejiggering that so that the rows notation appears closer to the top left of the image?

But everything else looks absolutely amazing

**Comment by kylebradbury** [comment]
Yes - I completely agree - I'll update this afternoon and share an new version!

**Comment by kylebradbury** [comment]
@nickeubank - I've updated this figure here: https://github.com/nickeubank/mids_coursera/blob/main/class_2/week_3/30_subsetting_matrices.ipynb

I also edited figure you mentioned the other day in lesson 2.3.10 on views and copies: https://github.com/nickeubank/mids_coursera/blob/main/class_2/week_3/10_views_and_copies.ipynb

Let me know if these look OK to you or if additional edits would be helpful.

**Comment by nickeubank** [comment]
Gorgeous as always!
